app/views/pages/_publishing_from_rss_guide_text.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="container article" id="editor-help-guide">
+  <div class="title help-guide-title">
+    <h1>Publishing from RSS Guide 📰</h1>
+  </div>
+  <div class="body">
+    <h2>Front Matter</h1>
+    <p>Our RSS reader converts some tags from the RSS items to tags for the Jekyll front matter:</p>
+    <ul>
+      <li><code>&lt;title&gt;</code>: <strong>title</strong></li>
+      <li><code>&lt;category&gt;</code>: <strong>tags</strong><br />Seperate tags by a colon.<br />Since only 4 tags can be used in a post on dev.to, the first four tags will be used from the RSS item.</li>
+      <li><code>&lt;link&gt;</code>: <strong>canonical_url</strong><br />Only if the "Mark the RSS source as canonical URL by default" option is enabled in the Publishing from RSS settings.</li>
+    </ul>
+
+    <h2>Article Content</h2>
+    <p>The HTML in the <code>&lt;description&gt;</code>, <code>&lt;summary&gt;</code>, or <code>&lt;description&gt;</code> tag will be converted to Markdown.</p>
```

Comment:
The priority order is content, summary, and then description.
https://github.com/thepracticaldev/dev.to/blob/f400b42e38504fb9464164b884e6a24021e1fe03/app/labor/rss_reader.rb#L128-L130

But the way Feedjira parses those items depends on the type of feed being used.

For example, in an Atom feed, only the content and summary tags are being used. See [here](https://github.com/feedjira/feedjira/blob/b91ff6feb9dd3d75c81edc9970f9c025d1dc964c/lib/feedjira/parser/atom_entry.rb#L8-L12).

In a RSS feed, the content tag is parsed from `content:encoded`, and the description tag is actually parsed from `summary` instead. See [here](https://github.com/feedjira/feedjira/blob/b91ff6feb9dd3d75c81edc9970f9c025d1dc964c/lib/feedjira/parser/rss_entry.rb#L13-L14)

I'm not sure what the best way to address this.

Reply:
Fixed usage order